pages/getting-started/installation.md

```diff
@@ -17,24 +17,18 @@ https://docs.conda.io/projects/conda/en/latest/user-guide/install/
 3. Activate the conda environment:<br/>
 `conda activate fisher`
 4. Add the Bioconda, Conda-Forge, & PhyloFisher Anaconda Cloud channels to your channels:<br/>
```

Comment:
Are they still adding the Phylofisher channel? Looks like you removed this step.

Reply:
Users won't need to add it anymore since PF has been moved into bioconda

Reply:
So should line 19 be edited to reflect this?

Reply:
Yeah for sure. I'll remove it now